src/testSetup.ts

```diff
@@ -701,8 +737,8 @@ const _testSetup = (sinon?: SinonStatic): TestContext => {
  *    $$.stubAliases({ 'myTestAlias': 'user@company.com' });
  *
  *    // Will use the contents set above.
- *    const username = (await StateAggregator.getInstance()).aliases.resolveUsername('myTestAlias');
- *    expect(username).to.equal('user@company.com');
+ *    const username = (await StateAggregator.getInstance()).aliases.resolveUsername("myTestAlias");
```

Comment:
fine with these changes, but should we keep these commented out UTs? Why not remove them?

Reply:
let me check

Reply:
@WillieRuemmele looks like jsdoc to me. I'll fix the `"`